src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,12 @@
+@import "./utils/_variables.scss";
+
+.catalog {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 4*$card-width+3*$card-margin-right;
```

Comment:
do not hardcome width on main tag, let`s better add one more wrapper inside of it
it will help to fix some failed tests
<img width="1286" alt="image" src="https://user-images.githubusercontent.com/66749919/207926482-8b50dab6-5e9c-415f-9b57-d9bd2f276824.png">